Makefile

```diff
@@ -247,11 +243,12 @@ dev-update-oss-components: cmd/draconctl/bin
 		--chart-version $(DRACON_VERSION) \
 		--name $(DRACON_OSS_COMPONENTS_NAME) \
 		./components
-	@helm upgrade $(DRACON_OSS_COMPONENTS_NAME) \
+	helm upgrade $(DRACON_OSS_COMPONENTS_NAME) \
 		./$(DRACON_OSS_COMPONENTS_NAME)-$(DRACON_VERSION).tgz \
 		--install \
 		--namespace dracon \
-		--set container_registry=kind-registry:5000/ocurity/dracon
+		--set container_registry=kind-registry:5000/ocurity/dracon \
+		--values ./deploy/deduplication-db-migrations/values/dev.yaml
```

Comment:
why are we passing the `deduplication-db-migrations` to the OSS components? The migrations container isn't even part of this chart?

Reply:
the oss-components have a strong dependency on the migrations since the deduplication db needs the same db values, instead of modifying the draconctl package command to include all of this and thus trying to work around Helm, we share values between the two